### ISSUE TITLE
Typo in gnt-cluster upgrade 

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#1071]
Originally reported of Google Code with ID 1015.

```
What software version are you running? Please provide the output of "gnt-
cluster --version", "gnt-cluster version", and "hspace --version".

gnt-cluster (ganeti v2.11.6) 2.11.6

Software version: 2.11.6
Internode protocol: 2110000
Configuration format: 2110000
OS api version: 20
Export interface: 0
VCS version: (ganeti) version v2.11.6

hspace (ganeti) version v2.11.6
compiled with ghc 7.4
running on linux x86_64

<b>What distribution are you using?</b>

Debian 7.5 amd64

<b>What steps will reproduce the problem?</b>
1. run gnt-cluster upgrade --to 2.11

2. during output:

     Running post-upgrade hooks
>> Unpasuing the watcher. <<
     Verifying cluster.

<b>What is the expected output? What do you see instead?</b>

Should read "Unpausing" instead of "Unpasuing"


```
Originally added on 2015-01-03 20:12:51 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2015-01-12 10:19:53 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit f3aff42bc2fafbd7049389ebc08be392248c3b25
Author: Aaron Karper <akarper@google.com>
Date:   Wed Jan 7 08:26:29 2015 +0100

    Fix typo in gnt_cluster output
    
    Fixes issue #1015
    
    Signed-off-by: Aaron Karper <akarper@google.com>
    Reviewed-by: Helga Velroyen <helgav@google.com>
    
    Cherry-picked from 19a5c6c351f815e89afe948897897a3eb14d29c3
    
    Signed-off-by: Klaus Aehlig <aehlig@google.com>
    Reviewed-by: Hrvoje Ribicic <riba@google.com>

diff --git a/lib/client/gnt_cluster.py b/lib/client/gnt_cluster.py
index af3f407..4d5af83 100644
--- a/lib/client/gnt_cluster.py
+++ b/lib/client/gnt_cluster.py
@@ -1984,7 +1984,7 @@ def _UpgradeAfterConfigurationChange(oldversion):
   if not _RunCommandAndReport([pathutils.POST_UPGRADE, oldversion]):
     returnvalue = 1
 
-  ToStdout("Unpasuing the watcher.")
+  ToStdout("Unpausing the watcher.")
   if not _RunCommandAndReport(["gnt-cluster", "watcher", "continue"]):
     returnvalue = 1
 

```
Originally added on 2015-01-12 10:57:41 +0000 UTC.

Changed State: Fixed
Added to Milestone: Release2.10

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released as part of 2.14.0
```
Originally added on 2015-06-02 10:40:20 +0000 UTC.

Changed State: Released
